hummingbird/ml/operator_converters/sklearn/linear.py

```diff
@@ -86,5 +86,7 @@ def convert_sklearn_linear_regression_model(operator, device, extra_config):
 register_converter("SklearnLinearRegression", convert_sklearn_linear_regression_model)
 register_converter("SklearnLogisticRegression", convert_sklearn_linear_model)
 register_converter("SklearnLinearSVC", convert_sklearn_linear_model)
+register_converter("SklearnLinearSVR", convert_sklearn_linear_regression_model)
```

Comment:
add these 2 new ones to the comments at the top of `convert_sklearn_linear_regression_model` line 20

Reply:
Ah yes!